pkg/helpers/array/array.go

```diff
@@ -1,4 +1,6 @@
-package helpers
+// Copyright (c) 2023 ScyllaDB
+
+package array
```

Comment:
`array` in Go is a slice with fixed size. This package contains functions working on slices, not arrays.  I would suggest to rename package and functions to use correct type name. 
Name of the package should be plural to be consistent with similar packages `bytes`, `strings` from stdlib.

Reply:
I agree, changed